src/main.cs

```diff
@@ -119,9 +119,14 @@ public class AboutProgramm : IEplAction
 
         public bool Execute(ActionCallingContext ctx)
         {
-            MessageBox.Show($"Версия надстройки - {GetVersion()}", 
-                "Версия надстройки", MessageBoxButtons.OK, 
-                MessageBoxIcon.Information);
+            string version = $"Версия надстройки - {GetVersion()}\n";
+            string license = "Проект распространяется под лицензией - MIT";
```

Comment:
```suggestion
            string license = "Проект распространяется под лицензией - MIT.";
```

Предлагается добавить точку в конце.